addons/binding/org.openhab.binding.tankerkoenig/src/main/java/org/openhab/binding/tankerkoenig/handler/WebserviceHandler.java

```diff
@@ -139,6 +139,13 @@ public void updateStationData() {
                 // if the result is not OK, no updates are done and the status of the Bridge goes to OFFLINE!
                 updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR,
                         "Empty return or no internet connection");
+                // if the Bridge goes OFFLINE, all Stations will go OFFLINE as well
+                for (Thing thing : getThing().getThings()) {
+                    StationHandler tkh = (StationHandler) thing.getHandler();
+                    tkh.updateStatus(ThingStatus.OFFLINE);
+                }
+                // and the scheduled job gets stopped!
+                pollingJob.cancel(true);
```

Comment:
IMHO  this will only be called if there was a reply from tankerkonig and this result is not "OK".  In this case there is something wrong with the request (one example is that the API-key got banned). All those reasons (are there more then one??) should be treated before new requests are sent (tankerkeonig asks for such a behaviour in its docs).
All other reasons for NO reply are treated elsewhere ( like no response due to no connection) and do not stop the polling.